server/src/main/java/org/opensearch/index/mapper/DateFieldMapper.java

```diff
@@ -331,6 +331,11 @@ public DateFieldMapper build(BuilderContext context) {
             Long nullTimestamp = parseNullValue(ft);
             return new DateFieldMapper(name, ft, multiFieldsBuilder.build(this, context), copyTo.build(), nullTimestamp, resolution, this);
         }
+
+        @Override
```

Comment:
Can we make metricSupported true for docCount mapper as well, its internal but just to denote that its used as metric.